homeassistant/bootstrap.py

```diff
@@ -73,10 +79,6 @@
     "cloud",
     # Ensure supervisor is available
     "hassio",
```

Comment:
I think this one needs to move with the frontend, mainly because it provides means to debug things as well (e.g., an add-on that provides access to logs or access to fix "stuff").

Reply:
I tried that first, but then the logging prefs gets lost for recorder if recorder moves along faster than logger setup :(

Reply:
I will test moving `hassio` as well soon

Reply:
working as expected. verified no events are fired from `hassio` so it should be fine.

Reply:
`hassio` provides entities since 2021.4, thus can fire events.

Reply:
There is a small risk that the recorder won't subscribe to the `async_initialize` before the first event is fired, but that seems worth the tradeoff to get hassio running sooner in case something goes wrong with startup later

Reply:
I could go either way on this one but I definitely see the value in having `hassio` up and running without waiting for the database upgrade in case something goes wrong.

Reply:
Came up with an alternate solution where the recorder does its initial setup to make sure it installs the event listener and then finishes up as soon as the frontend gets loaded then creates a tracked task to make sure its done before stage 1.

No changes to bootstrap and recorder doesn't block stage 1 from starting only stage 2.

Reply:
Well I had to add a check in bootstrap. I was really hoping to have it not know about it but we have to wait otherwise the events back up enough during a long database upgrade that we could risk exhausting the available system memory.